homeassistant/components/wolflink/manifest.json

```diff
@@ -5,6 +5,6 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/wolflink",
   "iot_class": "cloud_polling",
-  "loggers": ["wolf_smartset"],
-  "requirements": ["wolf-smartset==0.1.11"]
+  "loggers": ["wolf_comm"],
+  "requirements": ["wolf-comm==0.0.3"]
```

Comment:
Please add a license to the library.

Reply:
We also recommend accepting a common client session for the client library. Eg when using httpx it's good to accept an instance of `httpx.AsyncClient`. This way Home Assistant will benefit from connection pooling.

https://github.com/home-assistant/core/blob/8da07f1bf24d4b46f9c25f0aeebfb95c17a11ef4/homeassistant/helpers/httpx_client.py#L34-L37

Reply:
@MartinHjelmare I created a new version of wolf_comm (0.0.04) containing a LICENSE.txt. Thanks for the hint regarding connection pooling. I will incorporate in the next version. For now, I would like to get my first pull request done ^^